### ISSUE TITLE
Tuning emoji model parameters w/new methods

### ISSUE THREAD

**Issue by n8stringham** [n8stringham/gensim-evaluations#4]
Created the test set for emojis and evaluated with OddOneOut and Topk. Started looking through the emoji2vec github repo to figure out how their models are trained. Seems like I should be able to adjust the code to use our eval methods during training, but I'll need to look into it more.

**Comment by n8stringham** [comment]
Cloned the emoji2vec repo and started working with the code. Had to do a little debugging, but was able to get it running. I also modified `grid_search.py` to take the learning rate as a parameter. Next I'm gonna work on getting the code to evaluate using OddOneOut and Topk.
